cmd/fleetctl/get.go

```diff
@@ -120,7 +120,7 @@ func printSecret(c *cli.Context, secret *fleet.EnrollSecretSpec) error {
 	return printSpec(c, spec)
 }
 
-func printHost(c *cli.Context, host *fleet.Host) error {
+func printHost(c *cli.Context, host interface{}) error {
```

Comment:
Is `printHost` used from anywhere else such that this needs to be `interface{}`? Maybe it can just be changed to `*Fleet.HostResponse`?

Reply:
At the moment, `printHost` is only used in this one case. It makes sense to me to use `*Fleet.HostResponse`.

Reply:
the only reason I might be hesitant to use `HostResponse` is because it actually comes from package `service` so we need to introduce that package as a dependency:

i.e. `"github.com/fleetdm/fleet/v4/server/service"` in the imports section. We could move `status` down into the Host struct if that is preferable?

Reply:
`package service` is already a dependency due to `clientFromCLI` (https://github.com/fleetdm/fleet/blob/main/cmd/fleetctl/api.go#L60), and the `GetHosts` function we call from `service.Client`.

Arguably it may be more correct for `clientFromCLI` to return `fleet.Service` rather than `*service.Client` -- but I think we'd want to address that separately if we decide it's an issue.

cc @chiiph

Reply:
> Arguably it may be more correct for `clientFromCLI` to return `fleet.Service` rather than `*service.Client`

As in, the Service interface but implemented with a REST request rather than just going to the datastore? What would we be trying to solve with that approach?